### DISCUSSION TITLE
NuGet package author is wrong?

### DISCUSSION THREAD

**Discussion by jp2masa** [AvaloniaUI/Avalonia#10214] [General]
Only noticed now that the NuGet package author is the same as the package name, e.g. in `Avalonia.Desktop` the author is `Avalonia.Desktop`. The last correct version was 0.6.1 where the author was "Avalonia Team". Should this be fixed?

**Comment by maxkatz6** [comment]
Yes, seems like Authors information is missing here https://github.com/AvaloniaUI/Avalonia/blob/master/build/SharedVersion.props
